packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -95,6 +95,12 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    */
   scrollSelectedToTop?: boolean;
 
+  /**
+   * Should the combo box list be virtualized? Useful for combo boxes that contain a large number of items
+   * @default false
+   */
+  virtualized?: boolean;

```

Comment:
This is great! But it might be worth splitting into `VirtualizedComboBox`. That way we keep the bundle size of ComboBox minimal, and only pull List in when needed. What do you think?

Adding @christiango and @cschleiden here as well.

Reply:
That sounds reasonable to me, VirtualizedComboBox can extend from ComboBox and I can make the scrollTo function protected